pkg/resource/acl/custom_set_output.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package acl
+
+import (
+	"context"
+
+	svcsdk "github.com/aws/aws-sdk-go/service/memorydb"
+
+	svcapitypes "github.com/aws-controllers-k8s/memorydb-controller/apis/v1alpha1"
+	"github.com/aws-controllers-k8s/memorydb-controller/pkg/events"
+)
+
+func (rm *resourceManager) customDescribeACLsSetOutput(
+	ctx context.Context,
+	r *resource,
+	resp *svcsdk.DescribeACLsOutput,
+	ko *svcapitypes.ACL,
+) (*svcapitypes.ACL, error) {
```

Comment:
Since we are modifying `ko` in these methods, we shouldn't need to return a new `ACL` type. We can just return `error`

Reply:
@RedbackThomson in the operation "Describe", set_output_custom_method generates `ko, err = rm.xxxx` by code generator. If I delete ko in the return value, sdk.go cannot compile.

Reply:
Oh I see. I guess this is fine to leave, for now. It is a bit confusing that we're returning the same object, but it makes more sense than copying it and then returning a modified copy.